### PR TITLE
Updated the settings for creation a instance. Fixes #153

### LINKED ISSUE #153: Fix automation for RHV according to RHV4.1
- [ ] change url in automation from` https://RHVM-HOS/api/` to  `https://RHVM-HOST/ovirt-engine/api/`

- [ ] Change the hardcoded datacenter and cluster name

- [ ] New RHV has no quota defined , remove it from automation

### PR DESCRIPTION
**Pull request by san7ket**
Fixes #153

### CODE REVIEW

File: upgrade/helpers/rhevm.py [hunk 1]
```diff
@@ -70,24 +67,33 @@ def create_rhevm_instance(instance_name, template_name, datacenter='Default',
...
     instance.
     """
+    if not datacenter:
+        datacenter = os.environ.get('RHEV_DATACENTER')
+    if not cluster:
+        cluster = os.environ.get('RHEV_CLUSTER')
     rhevm_client = get_rhevm_client()
     template = rhevm_client.templates.get(name=template_name)
     datacenter = rhevm_client.datacenters.get(name=datacenter)
-    quota = datacenter.quotas.get(name=quota)
```
Comment (sghai) [inline upgrade/helpers/rhevm.py]: so quota is not required now ?

Comment (san7ket) [reply upgrade/helpers/rhevm.py]: No, we had it defined it in old setup , not in new as everyone uses single account

Comment (sghai) [reply upgrade/helpers/rhevm.py]: cool

**Comment by sghai** [comment]
ACK

**Pull request merged**